app/views/thredded/messageboards/index.html.erb

```diff
@@ -8,7 +8,10 @@
       <%= view_hooks.messageboards_index.list.render self, groups: @groups do %>
         <% @groups.each do |group| %>
           <% if group.name.present? %>
-            <h3 class="thredded--messageboards-group--title"><%= group.name %></h3>
+            <h3 class="thredded--messageboards-group--title">
+              <%= link_to group.name,
+                          show_messageboard_group_path(group.id) %>
```

Comment:
We use "slugs" for human-readable URLs throughout.
Let's think what URL format would work best here. Perhaps "/g/<slug>"?